system-int-test/test/cmr/system_int_test/ingest/collection_ingest_test.clj

```diff
@@ -607,7 +607,7 @@
        :iso-smap ["Line 1 - cvc-elt.1: Cannot find the declaration of element 'XXXX'."]))
 
 (deftest ingest-umm-json
-  (let [json (umm-spec/generate-metadata test-context exc/example-collection-record :umm-json)
+  (let [json (umm-spec/generate-metadata test-context exc/curr-ingest-ver-example-collection-record :umm-json)
```

Comment:
1. The name of the record: call it current will become obsolete soon once 1.10 becomes current.
2. Seems that we want to use the record that contains the new fields to do the ingest and other testings.  Not sure why it's specifically using the version 1.9 data record and now it's hardcoded to 1.9 version and we will always use this version of test data?
3. Also, do we have any example data files that contain the newly added fields in GetService and GetData? If not, we need to add one, to make the round-trip testing meaningful.

Reply:
Done, think I got them all.